src/adapter/etl-adapter-csv/tests/Flow/ETL/Adapter/CSV/Tests/Integration/CSVExtractorTest.php

```diff
@@ -220,4 +220,46 @@ public function test_loading_data_from_all_partitions() : void
                 ->toArray()
         );
     }
+
+    public function test_extracting_csv_with_more_than_1000_characters_per_line() : void
```

Comment:
Since this is an integration test, could you maybe use the entire Flow and full DSL here? 
It would increase the readability of the test. 
This test also looks like 2 tests, one that is failing and the other successful.